dev/build/windows/makecoq_mingw.sh

```diff
@@ -383,7 +383,7 @@ function build_conf_make_inst {
 function install_glob {
   # Check if any files matching the pattern exist
   if [ "$(echo $1)" != "$1" ] ; then
-    install -D -t $2 $1
+    install -D -t $2 "$1"
```

Comment:
This is still wrong, please look at my previous comment again.

Reply:
Fixed.